.buildkite/pipeline.yml

```diff
@@ -1,20 +1,59 @@
-env:
-  FORCE_COLOR: 1
-  GIT_TERMINAL_PROMPT: 0
+queues:
+  - &chromium_queue
+      runtimeType: chromiumbuild
+      queue: runtime
+  - &linux_agent_queue
+      <<: *chromium_queue
+      os: linux
+  - &windows_agent_queue
+      <<: *chromium_queue
+      os: windows
+  - &mac_agent_queue
+      <<: *chromium_queue
+      os: macos
 
-steps:
-  - label: "Build chromium - Linux"
-    key: "build-chromium-linux-x86_64"
-    timeout_in_minutes: 60
+plugins:
+  - &aws_sm_plugin
+    seek-oss/aws-sm#v2.3.1:
+      region: us-east-2
+      env:
+        BUILDEVENT_APIKEY: honeycomb-api-key
+        BUILDEVENT_BUILDKITE_API_TOKEN: buildkite-api-token-honeycomb-build-events
+  - &buildevents_plugin
+    replayio/buildevents#adb8a05: ~
```

Comment:
Ideally I would be able to express this as:

```
- &buildevents_plugins
  - seek-oss/aws-sm#v2.3.1:
      region: us-east-2
      env:
        BUILDEVENT_APIKEY: honeycomb-api-key
        BUILDEVENT_BUILDKITE_API_TOKEN: buildkite-api-token-honeycomb-build-events
  - replayio/buildevents#adb8a05: ~
```

and then reference the anchored list below like:

```
  - &linux_plugins
    plugins:
      - thedyrt/skip-checkout#v0.1.1:
          cd: /home/ubuntu/chromium/src
      <<: *buildevents_plugins
```
but alas, that isn't valid yaml.  instead each plugin needs its own anchor and those get mentioned individually in the os plugins lists.